infrateam_first_responder.md

```diff
@@ -54,7 +54,7 @@ Use the `sdr-deploy` script to deploy all infrastructure projects (with **import
 
 Note that you will need to be sure you can ssh into each of the VMs from wherever you are running the deploy script.
 
-- qa: deploy to qa with script
+- qa: deploy to qa first with script, then run infrastructure-integration-tests after deploy to qa.  Note as below that some tests mail fail but will pass if run individually.  You may also need to stay on the same "space" when tests are running if using a Mac to avoid focus issues with the browser.
```

Comment:
"mail fail" <-- typo.   I would rephrase second sentence:  "If on a Mac, you will get better results if you stay in the same "space" as the running tests (avoids focus issues with the browser)."

Reply:
thanks, changes made.